charts/neo4j-internal/values.yaml

```diff
@@ -1,20 +1,23 @@
+# https://github.com/neo4j/helm-charts/blob/dev/neo4j/values.yaml
+
 neo4j-standalone:
+
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 7474
+    runAsGroup: 7474
+    fsGroup: 7474
+    capabilities:
+      drop: ['ALL']
+    allowPrivilegeEscalation: false
+
   neo4j:
     labels:
       app.kubernetes.io/name: "neo4j-internal"
     resources:
       cpu: "0.5"
       memory: "2Gi"
 
-    securityContext:
-      runAsNonRoot: true
-      runAsUser: 7474
-      runAsGroup: 7474
-      fsGroup: 7474
-      capabilities:
-        drop: ['ALL']
-      allowPrivilegeEscalation: false
-
```

Comment:
So, i was getting kyverno alert allowPrivilegeEscalation despite it was correctly setted
looking at chart template
https://github.com/neo4j/helm-charts/blob/0744e2de758acce110e11e82909098c690280cb3/neo4j/templates/neo4j-statefulset.yaml#L61
we can see that it takes values from .Values.securityContext ...
So my recent changes had no effect and i did not saw it.

Reply:
looking at https://github.com/search?q=repo%3Aneo4j%2Fhelm-charts%20securityContext&type=code it seems that i can't add others parameters as those defined, so no way to add allowPrivilegeEscalation or capabilities

and indeed when i try here is the error message in postsync step of argocd
spec.template: Invalid value: core.PodTemplateSpec
...
field is immutable

and during the diff
Failed to compare desired state to live state: failed to calculate diff from cache: error calculating structured merge diff: error building typed value from config resource: .spec.template.spec.securityContext.allowPrivilegeEscalation: field not declared in schema

Reply:
so i will remove old security contexts i added which are useless
and i will add an exclude in kyverno for neo4j, 
do you agree @enys  ?

Reply:
i'm not sur i understand what you are saying. I do see       securityContext: {{ toYaml .Values.securityContext | nindent 8 }}

Reply:
Indeed it's easier when you know which securityContext parameter go to pod or container...
here are those for pod: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#podsecuritycontext-v1-core

it works fine now, thx for the help